dockerfiles/synapse/start.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
older versions of the Synapse docker image didn't include a /bin/bash, and this script works fine with regular sh, so I'm changing to that.